src/SupportedEntities.php

```diff
@@ -714,6 +714,23 @@ public static function getInfo() {
       if (!in_array($entity_info['civicrm entity name'], $api_entity_types)) {
         unset($civicrm_entity_info[$entity_type]);
       }
+
+      // Insert dblocale table names
+      $multilingual = \CRM_Core_I18n::isMultilingual();
+      if ($multilingual) {
+        // @codingStandardsIgnoreStart
+        global $dbLocale;
```

Comment:
is dbLocale different if accessing via different domains or different pages? 
Seems like if so, would have to clear the Drupal caches visiting the different domains for different languages for this to work. Each time switching language would need a cache clear? Can you provide some examples of when this value is different?